tests/testthat/test-pcens.R

```diff
@@ -1,21 +1,20 @@
 test_that("new_pcens creates object with correct structure", {
-  pdist_name <- "pgamma"
```

Comment:
in terms of depreciation we basically need to keep this test and show it still works with a depreciation

Reply:
added a test to the end which is explicitly dealing with the deprecation aspect; i *think* it covers relevant cases, and folks can focus updating that as with the upcoming deprecation stages